properties.schema

```diff
@@ -327,84 +337,84 @@
             "type": "string",
             "inputType": "ColourPicker",
             "default": "#FFFFFF",
-            "help": "Navigation background colour"
+            "help": "Sets the Navigation background colour."
```

Comment:
could the title be improved, negating the need for help text? e.g. "title": "Navigation background colour"